build.sbt

```
@@ -163,3 +163,27 @@ lazy val xml = crossProject(JSPlatform, JVMPlatform)
     Test / fork := false
   )
   .jsConfigure(_.enablePlugins(ScalaJSJUnitPlugin))
+  .nativeSettings(
+    scalaModuleMimaPreviousVersion := None, // No such release yet
+    // Scala Native cannot run forked tests
+    Test / fork := false,
+    libraryDependencies += "org.scala-native" %%% "junit-runtime" % nativeVersion % Test,
+    Test / scalacOptions += {
+      val log = streams.value.log
+      val retrieveDir = baseDirectory.value / "scala-native-junit-plugin-jars"
+      val lm = dependencyResolution.value
+      val cp = lm
+        .retrieve(
+          "org.scala-native" % s"junit-plugin_${scalaVersion.value}" % nativeVersion,
+          scalaModuleInfo = None,
+          retrieveDir,
+          log
+        )
+        .fold(w => throw w.resolveException, identity(_))
+      val jarPath = cp
+        .find(_.toString.contains(s"/org/scala-native/junit-plugin_${scalaVersion.value}/$nativeVersion/junit-plugin_${scalaVersion.value}-$nativeVersion.jar"))
```

Not sure this will work on Windows. Since this classpath is going to contain very few stuff (junit-plugin, scala-compiler, scala-reflect and scala-library, basically), you can make this much easier:
```suggestion
        .find(_.toString.contains("junit-plugin"))
```
and that will work on any OS.